src/VectorizeLoops.cpp

```diff
@@ -118,6 +118,25 @@ Interval bounds_of_lanes(const Expr &e) {
         } else if (is_negative_const(r->stride)) {
             return {r->base + last_lane_idx * r->stride, r->base};
         }
+    } else if (const LE *le = e.as<LE>()) {
```

Comment:
@rootjalex would also appreciate you checking this bounds-related logic. I wrote this code so I'm not well positioned to review it.

Reply:
Yeah, both of these cases follow the same logic as bounds inference, which has been verified.

@dsharletg is there a reason for the requirement that one of the two operands be a single point that I am missing? For inequalities, there are partial bounds that can be produced relatively easily.

Reply:
Nevermind, for the exact bounds requirement I think this is probably the best that can be done.

Reply:
I am not sure, @abadams added this. The logic looks like it would be correct without one side being a broadcast, but it wouldn't be exact, which the comment implies is a requirement.

Reply:
Yeah sorry, I should have read the top-level comment for `bounds_of_nested_lanes` to realize that this needed to be exact, my bad. I believe this is correct for the specification.